packages/schematics/angular/workspace/files/package.json.template

```diff
@@ -19,7 +19,6 @@
     "@angular/platform-browser": "<%= latestVersions.Angular %>",
     "@angular/platform-browser-dynamic": "<%= latestVersions.Angular %>",
     "@angular/router": "<%= latestVersions.Angular %>",
-    "core-js": "^2.5.4",
```

Comment:
how does the user get core-js on their machine if it's no longer in this package.json?

Reply:
It's a dependency of `build-angular` now; it was a hidden dependency before.  Moving it allows full control of the version preventing any potential mismatches; as well as allowing alternate polyfilling strategies moving forward without changing/breaking user's project dependencies.